lambdas/service/app.py

```diff
@@ -401,8 +399,6 @@ def _authenticate(self) -> Optional[OAuth2]:
                 else:
                     raise UnauthorizedError(header)
 
-    # FIXME: Return furl instance
-    #        https://github.com/DataBiosphere/azul/issues/3398
     def manifest_url(self,
```

Comment:
This fixme was not addressed, the method returns a str rather than a furl object.